web/src/compose.js

```diff
@@ -708,7 +706,7 @@ export function initialize() {
             $target_textarea = $(compose_click_target).closest("form").find("textarea");
         }
 
-        if ($(e.target).hasClass("has_popover")) {
+        if (!flatpickr.is_flatpickr_open) {
```

Comment:
Let's make the API `flatpickr.is_open()` to match the calling conventions for all of our other popovers. We likely also don't need the `is_flatpickr_open` variable -- can just return whether `current_flatpickr_instance` is `null` (and we should be sure to reset that variable when closing the `flatpickr` so it doesn't ever have a stale reference to a closed flatpickr object)